src/Controller/SantaController.php

```diff
@@ -31,17 +32,21 @@ class SantaController extends AbstractController
     private $twig;
     private $logger;
     private $applications;
+    private $statisticCollector;
 
-    public function __construct(RouterInterface $router, \Twig_Environment $twig, LoggerInterface $logger, array $applications)
+    public function __construct(RouterInterface $router, \Twig_Environment $twig, LoggerInterface $logger, array $applications, StatisticCollector $statistic)
     {
         $this->router = $router;
         $this->twig = $twig;
         $this->logger = $logger;
         $this->applications = $applications;
+        $this->statisticCollector = $statistic;
     }
 
     public function run(MessageDispatcher $messageDispatcher, Rudolph $rudolph, Request $request, string $application): Response
     {
+        $clientApplication = $application;
```

Comment:
why renaming this var?

Reply:
I delete the `$clientApplication` variable and I use `$application->getCode` instead